modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysInlineBoundariesTest.ts

```diff
@@ -336,4 +336,25 @@ describe('browser.tinymce.core.keyboard.ArrowKeysInlineBoundariesTest', () => {
       TinyAssertions.assertCursor(editor, [ 0, 0 ], 0);
     });
   });
+
+  context('Arrow keys when selection is not collapsed', () => {
+    it('TINY-8601: should move caret before selected inline boundary node when arrow left', () => {
+      const editor = hook.editor();
+      editor.setContent('<p>test <span class="mce-annotation">span</span> selection</p>', { format: 'raw' });
```

Comment:
I don't believe you need `format: 'raw'` here. Was there a reason that's being done?

Reply:
The only reason is that I tried to follow previous tests style.